spec/mentions_spec.rb

```diff
@@ -95,4 +95,21 @@ def para(content)
       expect(basic_post.output).to start_with(para(result.sub(default_src, mentions_src)))
     end
   end
+
+  context "when pulling the base from github metadata" do
+    let(:mentions_src) { "https://example.com" }
+    let(:config_overrides) do
+      {
+        "github" => { "mention_url_base" => mentions_src }
```

Comment:
Hm, I'm not convinced this data is ever available as a config. It's all in a Drop, only available to Liquid AFAIR.